<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="img_topic" default="/camera_front/image_color/compressed"/>
    <arg name="hm_frame" default="base_link"/>
    <arg name="linear_vels" default="[1.]"/>
    <arg name="angular_vels" default="[0.]"/>
    <arg name="total_sim_time" default="5."/>

    <!-- MonoDEM node -->
    <node name="monodem" pkg="monoforce" type="monodem_ros">
        <rosparam subst_value="true">
            img_topic: $(arg img_topic)
            hm_frame: $(arg hm_frame)
        </rosparam>
    </node>

    <!-- Diff Physics node -->
    <node name="diff_physics" pkg="monoforce" type="diff_physics" output="screen">
        <rosparam subst_value="true">
            hm_topic: /height_map/points
            hm_frame: $(arg hm_frame)
            linear_vels: $(arg linear_vels)
            angular_vels: $(arg angular_vels)
            total_sim_time: $(arg total_sim_time)
        </rosparam>
    </node>
</launch>
